src/app/component/ProductInformation/ProductInformation.component.js

```diff
@@ -87,6 +87,29 @@ export class ProductInformation extends PureComponent {
         );
     }
 
+    isHTMLWhiteSpaceOrUndefined(htmlString) {
+        if (!htmlString || htmlString.trim() === '') {
+            return true;
+        }
+
+        const parser = new DOMParser();
+        const document = parser.parseFromString(htmlString.trim(), 'text/html');
+
+        if (document.body.children.length === 0) {
+            return false;
+        }
+
+        // eslint-disable-next-line fp/no-let
+        let isWhiteSpace = true;
+        Array.from(document.body.children).forEach((element) => {
+            if (element.innerText !== '') {
+                isWhiteSpace = false;
+            }
+        });
+
+        return isWhiteSpace;
```

Comment:
1. This logic is not intuitive, please comment on it.
2. Avoid using let, it is prohibited by the ESLint rules. 
3. The same can be written by using `Array.prototype.find` method, please avoid mutating the scope inside of loops.